cachi2/core/package_managers/npm.py

```diff
@@ -124,23 +107,16 @@ def resolved_url(self) -> Optional[str]:
             # v2+ file dependency (or a workspace)
             elif self.path:
                 return f"file:{self.path}"
-            # v1 non-registry dependency
-            else:
-                return self.version
 
         return self._package_dict["resolved"]
 
     @resolved_url.setter
     def resolved_url(self, resolved_url: str) -> None:
         """Set the location where the package should be resolved from.
 
-        For v1/v2 package-lock.json `dependencies`, this will be the "resolved"
-        key for registry deps and the "version" key for non-registry deps.
-
         For v2+ package-lock.json `packages`, this will be the "resolved" key.
```

Comment:
Should we update all the docstrings where v2+ package-lock.json is specified since we support only v2+??

Reply:
I thought it would be better to keep it so we know that implementation is specific for those versions. But I'll remove it if you prefer not to have it there.

Reply:
The packages section has only v2+ `package-lock.json` that's why it sounds to me a little bit redundant. But if you want to keep it there to be more informative, I don't mind. Other thoughts @brunoapimentel  @chmeliik ?

Reply:
Same, I don't think we need it but I don't mind keeping it

Reply:
I also found it unnecessary since we're not supporting v1 anymore.

Reply:
changed the docstrings, ptal